### DISCUSSION TITLE
Smartthings motion sensors very slow.

### DISCUSSION THREAD

**Discussion by stratfin** [iklein99/homebridge-smartthings#133] [Q&A] [answered]
Since switching to this bridge I find my motion sensors very slow to update in automations. 

I have motion sensors to turn on lights on activity. In Smartthings native this would take 2 seconds max. With Smartthings lk bridge it takes 10 or more seconds.

I used to use the Smartthings bridge that made use of the IDE and the reaction time was acceptable - almost as quick as native Smartthings.

**Comment by iklein99** [comment, marked as answer]
That is because the plug-in has to poll Aeschylus device ap you have to wait for the next update.  Ima bout to release a version that uses a smart ap web hook which will result in near real time updates and will eliminate traffic resulting from the polling.  This capability will be available via a subscription. Sent from my iPhoneOn Jan 30, 2023, at 2:31 PM, Rob Eichhorn ***@***.***> wrote:﻿
Since switching to this bridge I find my motion sensors very slow to update in automations.
I have motion sensors to turn on lights on activity. In Smartthings native this would take 2 seconds max. With Smartthings lk bridge it takes 10 or more seconds.
I used to use the Smartthings bridge that made use of the IDE and the reaction time was acceptable - almost as quick as native Smartthings.

**Reply by stratfin** [reply]
Thanks - please keep me posted as to when you release this version.

**Comment by iklein99** [comment]
I am about ready to make this available, and I'm looking for one person to help me test it at the moment.  Are you interested?

**Reply by coryabooth** [reply]
Absolutely

**Comment by stratfin** [comment]
Yes I am.

Regards,

Rob Eichhorn
604-828-2210

**Reply by iklein99** [reply]
Rob,

Send me the following information and i'll send you an email with instructions and an access code:
1. Preferred user name (can be whatever)
2. email address

Send to iklein99@comcast.net
